assembl/alembic/versions/7ea03fbce9a8_create_vote_session_table.py

```diff
@@ -0,0 +1,69 @@
+from alembic import context, op
+from sqlalchemy import Column, Integer, ForeignKey
+from sqlalchemy.schema import UniqueConstraint
+import transaction
+
+"""create vote_session table
+
+Revision ID: 7ea03fbce9a8
+Revises: c3f8bc9c75d5
+Create Date: 2017-12-22 13:28:30.634303
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = '7ea03fbce9a8'
+down_revision = 'c3f8bc9c75d5'
+
+
+def LangStringId(column_name):
+    return Column(
+        column_name + '_id',
+        Integer(),
+        ForeignKey('langstring.id')
+    )
+
+
+def lang_strings_args(lang_strings_names):
+    args = [LangStringId(lang_string_name)
+        for lang_string_name in lang_strings_names]
+    
+    lang_strings_id_names = [name + "_id" for name in lang_strings_names]
+    
+    args.append(UniqueConstraint(*lang_strings_id_names))
```

Comment:
why is a unique constraint needed for langstrings?

Reply:
Good question, it was declared like that in the example I followed.
I can't think of a situation where the same langstring object would be used for two different fields tho